examples/diagnosis.py

```diff
@@ -23,6 +23,9 @@
     request.add_symptom('s_99', 'present')
     request.add_symptom('s_8', 'absent')
     request.add_symptom('s_25', 'present')
+    request.add_symptom('s_418', 'present', source='suggest')
+    request.add_symptom('s_18', 'present', source='predefined')
+    request.add_symptom('s_10', 'present', source='red_flags')
```

Comment:
Let's move that block under `request.add_symptom('s_98', 'present', source='initial')` line. Even though it's an example, form the flow perspective will make more sense.

Reply:
@akszydelko done